src/llama_stack_client/types/shared/photogen_tool_definition.py

```diff
@@ -4,7 +4,7 @@
 from typing_extensions import Literal
 
 from ..._models import BaseModel
-from ..rest_api_execution_config_param import RestAPIExecutionConfigParam
+from ..rest_api_execution_config import RestAPIExecutionConfig
```

Comment:
I think this will produce an error. Does `llama-stack-client` work?

Reply:
Let me add to the sync script to fix this

Reply:
fixed.